### PR TITLE
Theme changes

### PR DESCRIPTION
**Pull request by dainnilsson**

### CODE REVIEW

File: lib/app/views/app_page.dart [hunk 1]
```diff
@@ -238,9 +238,9 @@ class AppPage extends StatelessWidget {
                 },
                 icon: keyActionsBadge
                     ? const Badge(
-                        child: Icon(Icons.tune),
+                        child: Icon(Icons.more_vert_outlined),
```
Comment (AdamVe) [inline lib/app/views/app_page.dart:241]: 👍🏻

File: lib/oath/views/account_view.dart [hunk 2]
```diff
@@ -130,7 +130,7 @@ class _AccountViewState extends ConsumerState<AccountView> {
           final subtitle = helper.subtitle;
           final circleAvatar = CircleAvatar(
             foregroundColor: darkMode ? Colors.black : Colors.white,
-            backgroundColor: _iconColor(darkMode ? 300 : 400),
+            backgroundColor: _iconColor(darkMode ? 400 : 400),
```
Comment (AdamVe) [inline lib/oath/views/account_view.dart:133]: always 400

File: lib/piv/views/key_actions.dart [hunk 3]
```diff
@@ -43,110 +43,114 @@ Widget pivBuildActions(BuildContext context, DevicePath devicePath,
...
+                  leading: CircleAvatar(
+                    backgroundColor: theme.secondary,
+                    foregroundColor: theme.onSecondary,
+                    child: const Icon(Icons.laptop),
+                  ),
+                  onTap: () async {
+                    Navigator.of(context).pop();
+                  }),
+            ],
+            */
```
Comment (AdamVe) [inline lib/piv/views/key_actions.dart:151]: remove commented out code

File: lib/app/views/action_list.dart [hunk 4]
```diff
@@ -43,14 +43,14 @@ class ActionListItem extends StatelessWidget {
...
-    final (foreground, background) = switch (actionStyle) {
-      ActionStyle.normal => (theme.onSecondary, theme.secondary),
-      ActionStyle.primary => (theme.onPrimary, theme.primary),
-      ActionStyle.error => (theme.onError, theme.error),
-    };
+    // final (foreground, background) = switch (actionStyle) {
+    //   ActionStyle.normal => (theme.onSecondary, theme.secondary),
+    //   ActionStyle.primary => (theme.onPrimary, theme.primary),
+    //   ActionStyle.error => (theme.onError, theme.error),
+    // };
```
[the remark this answers is not in the document]
Comment (dainnilsson) [reply lib/app/views/action_list.dart:53]: @braathen - is this something we are going to re-introduce? Or should we remove the ActionStyle parameter alltogether?

[the remark this answers is not in the document]
Comment (braathen) [reply lib/app/views/action_list.dart:53]: mm, I wasn't sure at the time and felt the solution was nice to keep, but I guess we can always do it again if we feel there's a need for it.

**Review by braathen — APPROVED** [review]

**Pull request merged**